pkgs/by-name/st/streamrip/package.nix

```diff
@@ -2,10 +2,8 @@
   lib,
   python3Packages,
   fetchFromGitHub,
-
   ffmpeg,
 }:
-
 python3Packages.buildPythonApplication rec {
   pname = "streamrip";
   version = "2.0.5";
```

Comment:
Update the version according to https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning.

Should be `2.0.5-unstable-2025-01-05`.